entrypoint.sh

```diff
@@ -1,5 +1,7 @@
 #!/bin/sh
 
+set -o pipefail
```

Comment:
We would need to use bash 

```
#!/usr/bin/env bash
```

Reply:
@dgholz I’ll suggest using 

`set -euo pipefail` to prevent any other silent errors

Reply:
> We would need to use bash

I believe the `ruby:2.7-alpine` base image uses Busybox's shell, and it supports `set -o pipefail`. My testing was successful in the repo I liked from #27!

The composite action has to `!#/bin/bash` since the runner may not have `/bin/sh` set up to be a compatible shell; the GitHub-provided runners do, but custom self-hosted runners may not.

Reply:
> suggest using

set -euo pipefail to prevent any other silent errors

 Great idea! To be honest, I'm only really interested in the failure of one command in the script (`brakeman`), but this is a sensible suggestion and follows shell scripting best practices

I'd prefer to do that change separately, if you don't mind? This change is small enough as-is that it could be ported over to the composite action PR. I'd rather make it as easy as possible to land that PR so that I can get this change in.